### DISCUSSION TITLE
unable to connect a git repo to my argocd

### DISCUSSION THREAD

**Discussion by YoniSneOr** [argoproj/argo-cd#17578] [General]
Hi
My argoCD is on my GKE cluster  I can access it but when trying to add a git repo but it fails
try with SSH private key /HTTPS use/pass/PAT
none of them worked
how can I troubleshoot this?

**Comment by davidfalconego** [comment]
look for argocd-repo-server or argocd-server POD logs so it could be related to some firewall policies from one side
